openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1507,7 +1507,104 @@ It is up to the Credential Issuer whether to update both the signature and the c
+* Use cryptographic schemes that can provide non-correlation.
+
+Credential Issuers specifically SHOULD discard values that can be used in collusion with a Verifier to track a user, such as the Issuer's signature or cryptographic key material to which an issued credential was bound to.
+
+### Credential Offer
+
+The Privacy Considerations in Section 11.2 of [@!RFC9101] apply to the `credential_offer` and
+`credential_offer_uri` parameters defined in (#credential-offer).
+
+### Authorization Request
+
+The Wallet SHOULD NOT include potentially sensitive information in the Authorization Request,
+for example, by including clear-text session information as a `state` parameter value or encoding
+it in a `redirect_uri` parameter. A third party may observe such information through browser
+history, etc. and correlate the user's activity using it.
+
+## Identifier of the Credential Issuer
+
+Information in the credential identifying a particular Credential Issuer, such as a Credential Issuer Identifier,
+issuer's certificate, or issuer's public key may reveal information about the End-User.
+
+For example, when a military organization or a drug rehabilitation center issues a vaccine
+credential, verifiers can deduce that the owner of the Wallet storing such Credential is a
+military member or may have a substance use disorder.
+
+In addition, when a Credential Issuer issues only one type of Credential, it might have privacy implications,
+because if the Wallet has a Credential issued by that Issuer, its type and claim names can be
+determined.
+
+For example, if the National Cancer Institute only issued Credentials with cancer registry
+information, it is possible to deduce that the owner of the Wallet storing such Credential is a
+cancer patient.
+
+To mitigate these issues, a group of organizations may elect to use a common Credential Issuer,
+such that any credentials issued by this Issuer cannot be attributed to a particular organization
+through identifiers of the Credential Issuers alone. A group signature scheme may also be used
+instead of an individual signature.
+
+When a common Credential Issuer is used, appropriate guardrails need to be in place to prevent
+one organization from issuing illegitimate credentials on behalf of other organizations.
+
+## Custom URL Scheme
+
+There is a potential for leaking information about the Wallet to third parties when the
+Wallet reacts to a Credential Offer that was sent to a custom URL scheme that the Wallet
+supports. An attacker may send Credential Offers using different custom URL schemes, see if the
+Wallet reacts (e.g., whether the wallet retrieves Credential Issuer metadata hosted by an
+attacker's server), and, therefore, learn which Wallet is installed. To avoid this, the Wallet
+SHOULD require user interaction (or establish trust in the Issuer) before fetching any `credential_offer_uri ` or acting on the received Credential Offer.
```

Comment:
```suggestion
attacker's server), and, therefore, learn which Wallet is installed. To avoid this, the Wallet MAY
require user interaction (or establish trust in the Issuer) before fetching any `credential_offer_uri `
or acting on the received Credential Offer.
```